### DISCUSSION TITLE
Separation from lrzip

### DISCUSSION THREAD

**Discussion by pete4abw** [pete4abw/lrzip-next#71] [General]
At long last, there are no more file overlaps with `lrzip` and `lrzip-next`. All common files and man pages have been made unique in `lrzip-next`. All convenience symlinks, man pages, and programs are prefixed with **lrzn** consistently.

This was suggested by someone. I would like to attribute this enhancement, so let me know who you are!

Old Name | New Name
--- | ---
lrz | **lrzn**
lrz.1 | **lrzn.1**
lrzcat | **lrzncat**
lrzcat.1 | **lrzncat.1**
lrztar | **lrzntar**
lrztar.1 | **lrzntar.1**
lrzuntar | **lrznuntar**
lrzuntar.1 | **lrznuntar.1**
lrunzip | **lrznunzip**
lrunzip.1 | **lrznunzip.1**

Currently these changes are staged in the **[whats-next](https://github.com/pete4abw/lrzip-next/tree/whats-next)** branch for review and test. Tagged as v0.8.11

**Comment by demhademha** [comment]
Suggested in https://github.com/pete4abw/lrzip-next/issues/66#issuecomment-1052167531

**Reply by pete4abw** [reply]
I was trying to find that! Thank you for the push.
